docs/repositories/commits.md

```diff
@@ -32,14 +49,31 @@ The commit detail displays:
 
 Use the options in the cogwheel menu of each issue to [ignore and manage issues](issues.md#ignoring-and-managing-issues).
 
-![Expanded issue view](images/commits-detail-issue-actions.png)
+![Expanded issue view](images/issues-menu.png)
+
+## Duplication tabs
 
-The commit differences view allows you to review the lines of code changed in the commit.
+The **New Duplication** and **Fixed Duplication** tabs display the list of duplicated blocks that the commit created or fixed.
 
-![Commit differences view](images/commits-differences.png)
+![New Duplication and Fixed Duplication tabs](images/commits-tab-duplication.png)
+
+## Files tab
+
+The **Files** tab displays an overview of the code quality changes introduced by each file that was changed in the commit.
+
+![Files tab](images/commits-tab-files.png)
+
+## Diff tab
+
+The **Diff** tab displays the line differences in each file that was changed in the commit.
+
+![Diff tab](images/commits-tab-diff.png)
 
 ## Possible issues
 
+<!--NOTE
+    Consider moving this section to a dedicated page under the FAQs-->
+
 In some situations, Codacy may report either new or fixed **possible** issues on a commit or pull request, which means that the code analysis detected these issues in lines of code that weren't changed by that commit or pull request. This gives you awareness to how your changes may be affecting other parts of your code.
```

Comment:
If we are reading the documentation for the commits, should we be talking about the pull requests too? 

"issues on a commit or pull request" -> "issues on a commit"
"by that commit or pull request" -> "by that commit"

If the section is moved to a faq section then shared then i'd keep it I guess.

Reply:
Good feedback! :+1: 

@stefanvacareanu7 suggested that we move this info inside the "Issues tab" section, and I'll also take care of mentioning only commits on the Commits page and pull requests on the Pull requests page. See https://github.com/codacy/docs/pull/1010 if you're curious about how I'm planning to do this without essentially repeating the same information on both pages. :slightly_smiling_face: